test/integration/test-video-players-helper.js

```diff
@@ -256,12 +256,16 @@ export function runVideoPlayerIntegrationTests(
   }
 
   function cleanUp() {
```

Comment:
That's because any timeout in the iframe will still fire, causing all kinds of mayhem.